academy-of-management-review.csl

```diff
@@ -99,7 +102,14 @@
                 </date>
               </if>
             </choose>
-          </if>
+          </else-if>
+          <else-if variable="issued" match="any">
+            <choose>
```

Comment:
i understand why you need to test for issued. But why do you need to test for status? You should never need to test for a variable in a `choose` element with just one `if`. That's implicit -- does that make sense? Or what am I missing?